FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Josue Quiñones on 05/07/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class FeedImageMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+
+		var images: [FeedImage] {
+			return items.map { $0.image }
+		}
+	}
+
+	private struct Item: Decodable {
+		public let imageId: UUID
+		public let imageDesc: String?
+		public let imageLoc: String?
+		public let imageURL: URL
```

Comment:
`Item` is private so its properties should not be public, please remove `public` modifiers.

```suggestion
		let imageId: UUID
		let imageDesc: String?
		let imageLoc: String?
		let imageURL: URL
```